docs/website/versioned_docs/version-3.0.0/user-guides/quickstart.md

```diff
@@ -10,6 +10,8 @@ In this guide, we will be using odo to create a "Hello World" application.
 You have the option of choosing from the following frameworks for the quickstart guide:
 * Node.js
 * .NET
+* Java (Spring Boot)
+* Go
```

Comment:
Would it make sense to make this alphabetically chronological? Including the TabItem tags below as well.

Reply:
Ah, no. We want to focus on Node.js first as it's the most tested, then .NET, Java and lastly Go.

Reply:
Sure 👍🏾